File: protocol/src/main/java/spp/protocol/probe/error/MissingRemoteException.java
package spp.protocol.probe.error;

public class MissingRemoteException extends RuntimeException {

    public MissingRemoteException(String remote) {
        super(remote, null, true, false);
    }

    @Override
    public synchronized Throwable fillInStackTrace() {
        return this;
    }

    @Override
    public String toString() {
        return getLocalizedMessage();
    }
}

File: protocol/src/main/kotlin/spp/protocol/probe/error/MissingRemoteException.kt
package spp.protocol.probe.error

class MissingRemoteException(remote: String) : RuntimeException(remote, null, true, false) {
    @Synchronized
    override fun fillInStackTrace(): Throwable {
        return this
    }

    override fun toString(): String {
        return localizedMessage
    }
}
